packages/synapse-react-client/src/components/AccountLevelBadges/AccountLevelBadges.tsx

```diff
@@ -86,3 +86,5 @@ export function AccountLevelBadges({ userId }: AccountLevelBadgesProps) {
     </>
   )
 }
+
+export default AccountLevelBadges
```

Comment:
Added `default` export to many component files to simplify dynamic import syntax